test/doc.go

```diff
@@ -1,6 +0,0 @@
-// Package test contains integration tests with fake Slack and Kubernetes APIs.
```

Comment:
do you want to still address that? 
https://github.com/infracloudio/botkube/pull/622#discussion_r901612701

Reply:
Thanks, right 👍 Will do that